nss-day-cohort-48/rare-client-boastful-bulldogs
# Delete Posts

delete button on posts and my posts fully functional; only displaying for logged in user

#steps to test:
1. git fetch --all
2. git checkout 5hann (server and client)
3. create post if necessary; delete post in "my post" view
4. delete another post in all post view. 
5. watch network tab to ensure no errors/unnecessary fetch calls
